app/services/search_service.rb

```diff
@@ -160,16 +160,23 @@ def tagNearbyPeople(coordinates, tag, period = nil, sort_by = nil, order_directi
                          .distinct
 
     if tag.present?
-      user_locations = User.joins(:user_tags)
-                           .where('user_tags.value LIKE ?', tag)
-                           .where(id: user_locations.select("rusers.id"))
+      if field.present? && field == 'node_tag'
+        tids = Tag.where("term_data.name = ?", tag).collect(&:tid).uniq || []
+        uids = TagSelection.where('tag_selections.tid IN (?)', tids).collect(&:user_id).uniq || []
+      else
+        uids = User.joins(:user_tags)
+                   .where('user_tags.value LIKE ?', tag)
```

Comment:
Here is LIKE necessary? And if it is, should we use the % character? I'm not super familiar with LIKE outside the use of that character but just checking. Thanks!

Reply:
I think it would work the same way as `=` in this case (https://stackoverflow.com/questions/543580/equals-vs-like). But I changed it to `'user_tags.value = ?'`. And we don't want to use `%` character as we need the exact match.